cli/azd/internal/repository/app_init.go

```diff
@@ -464,6 +464,14 @@ func (i *Initializer) prjConfigFromDetect(
 				return config, err
 			}
 		}
+		// if java project no Dockerfile, add a default one
+		if svc.Language == appdetect.Java && svc.Docker == nil {
+			docker, err := appdetect.AddDefaultDockerfile(svc)
```

Comment:
If `AddDefaultDockerfile` only used in `app_init.go`, how about moving it to current package / file?

Reply:
This func is the part of `Docker`, and in the func body, it also refers to something of the `appdetect` package, so it would be better to place it into current `appdetect` package.

Reply:
Moved this func to `azd package` phrase.